light_node/src/main.rs

```diff
@@ -233,10 +233,19 @@ fn block_on_actors(
             .num_of_peers_for_bootstrap_threshold(),
     );
 
+    let mut actor_system_cfg = riker::load_config();
```

Comment:
@binier 
hmm, env.riker_threads is not used anywhere,
I would better do it like this, if it is set, then use it, else use cpu_get

```
    if env.riker_threads > 0 {
        actor_system_cfg
            .set("dispatcher.pool_size", env.riker_threads as i64)
            .unwrap();
    } else {
        let thread_count = env.riker_threads.min(num_cpus::get());
        actor_system_cfg
            .set("dispatcher.pool_size", env.riker_threads as i64)
            .unwrap();
    }
```

Reply:
`let thread_count = env.riker_threads.min(num_cpus::get());`

It is used bellow. We calculate minimum of available cores and `env.riker_threads`

If `riker_threads == 0`, then we don't need to override `dispatcher.pool_size` at all.

Reply:
@binier 
ahaaa, now I see, it calculates min